cime_config/tests.py

```diff
@@ -210,6 +210,7 @@
             "SMS_D_Ld1.T62_oEC60to30v3.DTESTM",
             "SMS_D_Ld3.T62_oQU120.CMPASO-IAF",
             "SMS_D_Ld1.ne30pg2_r05_EC30to60E2r2.WCYCL1850",
+            "SMS_Ln5.ne30pg2_ne30pg2.F2010-SCREAM-LR-DYAMOND2",
```

Comment:
I'd like to see a DP test as well.

Reply:
DP test added to the developer test suite.

Reply:
DP = "doubly-periodic" ?   Hard to tell from the name that its that kind of domain.